docs/data-sources/mlflow_model.md

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "MLflow"
+---
+# databricks_mlflow_model Data Source
+
+-> **Note** If you have a fully automated setup with workspaces created by [databricks_mws_workspaces](../resources/mws_workspaces.md) or [azurerm_databricks_workspace](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/databricks_workspace), please make sure to add [depends_on attribute](../index.md#data-resources-and-authentication-is-not-configured-errors) in order to prevent _default auth: cannot configure default credentials_ errors.
+
+Retrieves the settings of [databricks_mlflow_model](../resources/mlflow_model.md) by name.
+
+## Example Usage
```

Comment:
could you add examples of how this data source can potentially be used with other mlflow resources?

Reply:
@840 +1 to this as well.

Reply:
Added